src/Controls/src/Core/Entry.cs

```diff
@@ -22,6 +22,7 @@ public partial class Entry : InputView, IFontElement, ITextAlignmentElement, IEn
 		public new static readonly BindableProperty TextProperty = InputView.TextProperty;
 
 		public new static readonly BindableProperty TextColorProperty = InputView.TextColorProperty;
+		public new static readonly BindableProperty KeyboardProperty = InputView.KeyboardProperty;
```

Comment:
Was this never here before? Do we really need to add this for derived controls @pureween?